### PR TITLE
README update

### PR DESCRIPTION
**Pull request by noreenwu**
Clarified the npm install process for getting up and running on developing for blocks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,20 +44,38 @@ Install lerna (the tool we use to manage multiple packages):
 npm install -g lerna
 ```
 
+Cd into each package (blocks-base, blocks-docs, blocks-react) and run
```
Comment (MagdalenaHenke) [inline README.md]: Leave example comment.

File: README.md [hunk 2]
```diff
@@ -44,20 +44,38 @@ Install lerna (the tool we use to manage multiple packages):
...
+```
+npm install
```
Comment (MagdalenaHenke) [inline README.md]: Leave an example review

File: README.md [hunk 3]
```diff
@@ -44,20 +44,38 @@ Install lerna (the tool we use to manage multiple packages):
...
+```
+in each.
+
+
+Ensure that gatsby is running on your machine. If it isn't, try
+installing it globally:
+```
+npm install -g gatsby-cli
```
Comment (MagdalenaHenke) [inline README.md]: this looks great!

**Comment by noreenwu** [comment]
@MagdalenaHenke  please review

**Pull request merged**